src/Lesson2.java

```diff
@@ -138,31 +136,31 @@ public static boolean isSortedInAscendingOrder(int[] array) {
         return true;
     }
 
-    public static int[] isRandomSortedArray(int[] array) {
+    public static int[] randomSortedArray(int[] array) {
 
         int randomIndex;
 
         int[] arrayResult = new int[array.length];
         for (int i = 0; i < arrayResult.length; i++) {
-            randomIndex = (int) (Math.random() * (array.length - 1));
+            randomIndex = (int) (Math.random() * (array.length));
             arrayResult[i] = array[randomIndex];
-            array = isDeleteArrayElement(array, randomIndex);
+            array = deleteArrayElement(array, randomIndex);
         }
         return arrayResult;
     }
 
 
-    public static int[] isArrSorting(int[] array) {
+    public static int[] sortingArray(int[] array) {
         int[] resultArray = new int[array.length];
         for (int i = 0; i < resultArray.length; i++) {
-            resultArray[i] = isMinElementArray(array);
-            array = isDeleteArrayElement(array, dellIndex);
+            resultArray[i] = minElementArray(array);
+            array = deleteArrayElement(array, dellIndex);
```

Comment:
we are not using global variables
dellIndex should be visible only inside the method so we cant declare it outside